examples/systemcalls/systemcalls.c

```diff
@@ -16,7 +24,11 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
-
+    int ret_val = system(cmd);
+    if(ret_val == -1)
```

Comment:
@sankalp123321  think about what would happen if the cmd returned with an error code other than -1 here.

Reply:
You're right, I had to add the whole if else ladder for the error cases. I got late submitting. Will take care from assignment 4.

Reply:
Or you could simply add a reverse condition? If not zero maybe?